consensus/src/test/java/org/apache/iotdb/consensus/ratis/RatisConsensusTest.java

```diff
@@ -185,6 +185,21 @@ public void crashAndStart() throws Exception {
     doConsensus(servers.get(0), gid, 10, 210);
   }
 
+  @Test
+  public void oneMemberGroupChange() throws Exception {
+    List<Peer> original = peers.subList(0, 1);
+
+    servers.get(0).createPeer(gid, original);
+    doConsensus(servers.get(0), gid, 10, 10);
+
+    servers.get(1).createPeer(gid, peers.subList(0, 2));
+    servers.get(0).addPeer(gid, peers.get(1));
+    servers.get(0).removePeer(gid, peers.get(0));
```

Comment:
both `servers.get(0).removePeer(gid, peers.get(0));` and `servers.get(1).removePeer(gid, peers.get(0));` should be tested.

![image](https://user-images.githubusercontent.com/6756545/190441770-6484e205-ad86-40fd-8f97-514e321ecfae.png)

Reply:
done